### PR TITLE
Add SPDX header - batch 8

### PR DESCRIPTION
**Pull request by AndyScherzinger**
* Resolves: SPDX batch 8

## Summary

get some `apps/` done with SPDX headers

batch8

* files with copyright information: 9963 / 10961
* files with license information: 8725 / 10961

## TODO

- [x] add remaining files
- [ ] get it reviewed and merged

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
Any help on how to fix https://github.com/nextcloud/server/actions/runs/9274173027/job/25516154351?pr=45554 is highly appreciated 🙏

@susnux maybe?

File: apps/files/js/templates.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+/**
```
Comment (susnux) [inline apps/files/js/templates.js:1]: @AndyScherzinger i fear you have to use the dep5 file here to prevent ci error

Comment (AndyScherzinger) [reply apps/files/js/templates.js:1]: Alright, thanks

**Review by artonge — APPROVED** [review]

**Review by susnux — APPROVED** [review]

File: apps/files/js/templates.js (hunk 1 above)
Comment (susnux) [reply apps/files/js/templates.js]: (really need to unify files for public and internal so we can just drop handlebars :sweat_smile: )

**Pull request merged**